### PR TITLE
create word with options

### PR DESCRIPTION
**Pull request by MuhammadTamzid**

### CODE REVIEW

File: app/views/admin/words/_answer_fields.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%= answer_form.link_to_remove "Remove option" %>
```
Comment (sujoydatta) [inline app/views/admin/words/_answer_fields.html.erb]: i18n

File: app/views/admin/words/new.html.erb [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
       <%= f.label :content %>
       <%= f.text_field :content, class: "form-control" %>
 
+      <%= f.label :options %>
+
+      <%= f.fields_for :word_answers do |answer_form| %>
+        <%= render "answer_fields", answer_form: answer_form %>
+      <% end %>
+
+      <p><%= f.link_to_add "Add option", :word_answers %></p>
```
Comment (sujoydatta) [inline app/views/admin/words/new.html.erb]: i18n

**Comment by sujoydatta** [comment]
ok

File: app/models/word.rb [hunk 3]
```diff
@@ -9,4 +9,7 @@ class Word < ActiveRecord::Base
     presence: true,
     length: {maximum: 20},
     uniqueness: {case_sensitive: false}
+
+  accepts_nested_attributes_for :word_answers, allow_destroy: true,
+    reject_if: proc {|a| a[:content].blank?}
```
Comment (dieunb) [inline app/models/word.rb]: don't use short name

**Pull request merged**